BMM.UI.Android/Application/ViewHolders/GibraltarProjectBoxExpandedViewHolder.cs

```diff
@@ -52,17 +55,26 @@ private void SetItems(IBmmObservableCollection<IAchievementPO> achievements)
 
         ItemView.Post(() =>
         {
-            int imageSize = 64.DpToPixels();
+            int imageSize = ImageSize.DpToPixels();
             int imageCount = achievements.Take(4).Count();
             int layoutWidth = achievementsLayout.Width;
 
-            int totalImageWidth = imageSize * imageCount;
-            int remainingSpace = layoutWidth - totalImageWidth;
-            int spacing = remainingSpace > 0
-                ? remainingSpace / (imageCount - 1)
-                : 0;
+            int spacing;
+            if (DeviceInfo.Idiom == DeviceIdiom.Tablet)
```

Comment:
Is it smart to base the decision on that? Isn't it better to look at how many pixels are available?
That would probably work better on folding devices?

Reply:
looks like idiom takes the screen width into account: https://github.com/dotnet/maui/blob/main/src/Essentials/src/DeviceInfo/DeviceInfo.android.cs